.github/workflows/.smoke.yml

```diff
@@ -0,0 +1,58 @@
+# reusable workflow
+name: .smoke
```

Comment:
Why do we need a reusable workflow for smoke tests? It seems only called from `test-graphdriver`.

Reply:
Maybe instead we could just move this to a regular workflow named `test-smoke.yml` as it's not tied to graphdriver or snapshotter.

Reply:
I assume the idea is that if `smoke` fails, we can abort the run early. That's what the name suggests to me, anyway. So perhaps it should be called from both test-runs?